lms/services/blackboard_api/_schemas.py

```diff
@@ -54,7 +54,7 @@ class Meta:
 
         id = fields.Str(required=True)
         name = fields.Str(required=True)
-        groupSetId = fields.Str(required=True)
+        groupSetId = fields.Str(required=False, allow_none=True)
```

Comment:
@seanh You mention yesterday that this was a possibility, we don't care about groups with no groupset but the API will return them and we have to filter them out, good find :+1: